### ISSUE TITLE
Pytorch 2.1.0 pyg_lib not existed

### ISSUE THREAD

**Issue by LukeLIN-web** [pyg-team/pytorch_geometric#8172]
### 🐛 Describe the bug

https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html

```
pip install pyg_lib torch_scatter torch_sparse -f https://data.pyg.org/whl/torch-2.1.0+cpu.html

Looking in links: https://data.pyg.org/whl/torch-2.1.0+cpu.html
ERROR: Could not find a version that satisfies the requirement pyg_lib (from versions: none)
ERROR: No matching distribution found for pyg_lib
```
Same in https://data.pyg.org/whl/torch-2.1.0+cu118.html

### Environment

* PyG version: 2.3.1
* PyTorch version: 2.1.0
* OS:
* Python version:
* CUDA/cuDNN version:
* How you installed PyTorch and PyG (`conda`, `pip`, source): pip 
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by akihironitta** [comment]
@LukeLIN-web I believe it hasn't been released yet. Would you mind trying this link?
```
https://data.pyg.org/whl/nightly/torch-2.1.0+cpu.html
```

**Comment by rusty1s** [comment]
Just released `pyg-lib==0.3.0` :)
